### ISSUE TITLE
I want to train my dataset

### ISSUE THREAD

**Issue by Seonguke** [shariqfarooq123/AdaBins#23]
Hi 
As the title, I have some questions about train for my dataset.

1. what does 'focal' mean & How does that affect learning?
2. How much loss is required for a good output?

**Comment by shariqfarooq123** [comment]
> Hi 
> 
> As the title, I have some questions about train for my dataset.
> 
> 
> 
> 1. what does 'focal' mean & How does that affect learning?
> 
> 2. How much loss is required for a good output?
> 
> 
> 
> 

1. 'focal' values are the camera params and are not used in this paper. 
2. Your second question is very difficult to answer even in the whole context of deep learning. However, you can keep a validation set. Then lower the validation loss, the better.
